client/src/pages/customer-pages/FlightMap/index.js

```diff
@@ -62,6 +73,18 @@ class FlightMap extends Component {
           &nbsp;-&nbsp;
           <label>{destination}</label>
         </div>
+        <Map
+          markers={[
+            { position: positions[origin] },
+            { position: positions[destination] }
+          ]}
+          zoom={5}
+          center={{ lat: 9.072264, lng: 7.491302 }}
+          containerElement={
+            <div style={{ height: 180 + "px", width: 300 + "px" }} />
```

Comment:
You don't need to specify `px` when assigning numbers to style properties. React adds `px` as default. Although you'll have to add suffix if it was `%` or `vw` etc...

Reply:
done